integrations/crowdnode/src/main/res/layout/fragment_portal.xml

```diff
@@ -113,7 +113,7 @@
                 android:id="@+id/deposit_btn"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:foreground="?attr/selectableItemBackground"
+                android:background="?attr/selectableItemBackground"
```

Comment:
There is also a small UI issue with this button because of the `foreground` property. It was used to make a selected button look good when there is the "deposit more.." badge present.

Now the "deposit more..." badge has alpha in its color and so we can change this to `background` and it'll look fine.